src/pipecat/services/xtts.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import aiohttp
+
+from typing import AsyncGenerator
+
+from pipecat.frames.frames import AudioRawFrame, ErrorFrame, Frame
+from pipecat.services.ai_services import TTSService
+
+from loguru import logger
+
+import requests
+
+import numpy as np
+import resampy
+
+#####
+## The server below can connect to XTTS through a local running docker
+##
+## Docker command: $ docker run --gpus=all -e COQUI_TOS_AGREED=1 --rm -p 8000:80 ghcr.io/coqui-ai/xtts-streaming-server:latest-cuda121
+## 
+## You can find more information on the official repo: https://github.com/coqui-ai/xtts-streaming-server
+####
+
+SERVER_URL = 'http://localhost:8000'
+
+class XTTSService(TTSService):
+
+    def __init__(
+            self,
+            *,
+            aiohttp_session: aiohttp.ClientSession,
+            voice_id: str,
+            language: str,
+            **kwargs):
+        super().__init__(**kwargs)
+
+        self._voice_id = voice_id
+        self._language = language
+        self._aiohttp_session = aiohttp_session
+        self.STUDIO_SPEAKERS = requests.get(SERVER_URL + "/studio_speakers").json()
```

Comment:
`self._studio_speakers` :pray:

Reply:
The `/studio_speakers` is a default path on xtts... maybe just passing the base_url would solve, what you think?

Reply:
Ah, I got it. The uppercase. My bad. fixing it.